### PR TITLE
Placeholders for Select

### PR DESCRIPTION
**Pull request by derpdead**
# Description

Add placeholder for Select and implement it across application

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Unit Tests
- [ ] e2e Test

# Checklist:

- [x] I have read the contribution requirements and fulfil them.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: modules/@ergonode/attributes/src/components/Forms/AttributeForm.vue [hunk 1]
```diff
@@ -46,7 +46,22 @@
...
+                        <DropdownPlaceholder
+                            v-if="isVisible"
+                            :title="placeholder.title"
+                            :subtitle="placeholder.subtitle">
+                            <template #action>
+                                <Button
+                                    title="GO TO ATTRIBUTE GROUPS"
+                                    :size="smallSize"
+                                    :disabled="!isAllowedToUpdate"
+                                    @click.native="onNavigateToAttributeGroups" />
```
Comment (gygoo) [inline modules/@ergonode/attributes/src/components/Forms/AttributeForm.vue:60]: it would be more readable
`@click.native="onNavigateToAttributeGroups()" `

Comment (derpdead) [reply modules/@ergonode/attributes/src/components/Forms/AttributeForm.vue:60]: We do not allow to use parentheses with no arguments inside v-on binding

File: modules/@ergonode/categories/src/extends/components/Forms/ProductFormCategory.vue [hunk 2]
```diff
@@ -45,12 +68,31 @@ export default {
...
+        },
     },
     methods: {
         ...mapActions('product', [
             '__setState',
         ]),
-        setCategoriesValue(value) {
+        onNavigateToCategories() {
+            this.$router.push({
+                name: 'categories-grid',
```
Comment (gygoo) [inline modules/@ergonode/categories/src/extends/components/Forms/ProductFormCategory.vue:92]: I would create a configuration file and import that as constants throughout the app 
also in router 

`export const CATEGORIES_GRID = 'categories-grid';

export default {
    CATEGORIES_GRID
};


this.$router.push({
                name: CATEGORIES_GRID,
})
`

Comment (derpdead) [reply modules/@ergonode/categories/src/extends/components/Forms/ProductFormCategory.vue:92]: That's good idea. We will create separate ticket for this, cuz it should be implemented across whole app.

File: modules/@ergonode/ui/src/components/Select/Dropdown/SelectDropdown.vue [hunk 3]
```diff
@@ -4,110 +4,95 @@
...
-                                        </ListElementDescription>
-                                    </template>
-                                </slot>
+                                    :index="index" />
                             </template>
-                        </ListElement>
+                        </DropdownListElement>
+                        <DropdownPlaceholder
+                            v-if="isSearchPlaceholderVisible"
+                            title="No results"
```
Comment (gygoo) [inline modules/@ergonode/ui/src/components/Select/Dropdown/SelectDropdown.vue:43]: to computed?

**Review by gygoo — APPROVED** [review]

**Pull request merged**